include/El/core/environment/decl.hpp

```diff
@@ -265,6 +265,24 @@ DEBUG_ONLY(
     typedef CallStackEntry CSE;
 )
 
+DEBUG_ONLY(
+    void OpenLog( const char* filename );
+
+    std::ostream & LogOS();
+
+    void LogStr( std::string str );
```

Comment:
Is there a reason to still call this `LogStr` instead of `Log`? C++ supports overloading, so the `Str` is implied by the input argument.